app/editor/src/store/hooks/lookup/useLookup.ts

```diff
@@ -219,14 +219,16 @@ export const useLookup = (): [ILookupState, ILookupController] => {
           'lookup',
         );
       },
-      getTopics: async () => {
+      getTopics: async (refresh?: boolean) => {
```

Comment:
strangely, we have this refresh parameter in the declaration of the method but then we dont use it.  This implementation seems to make sense.